contracts/test/StakeTable.t.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: Unlicensed
+
+/* solhint-disable contract-name-camelcase, func-name-mixedcase, one-contract-per-file */
+
+pragma solidity ^0.8.0;
+
+// Libraries
+import "forge-std/Test.sol";
+import { BN254 } from "bn254/BN254.sol";
+import { BLSSig } from "../src/libraries/BLSSig.sol";
+import { EdOnBN254 } from "../src/libraries/EdOnBn254.sol";
+import "../src/interfaces/IStakeTable.sol";
+
+// Token contract
+import { ExampleToken } from "../src/ExampleToken.sol";
+
+// Target contract
+import { StakeTable as S } from "../src/StakeTable.sol";
+
+contract StakeTable_keyRegister_Test is Test {
```

Comment:
nit: name should simply be `StakeTable_register_Test is Test` where the middle one matches exactly the corresponding target function. 

maybe skim through the[ guidance here](https://book.getfoundry.sh/tutorials/best-practices#general-test-guidance).

Reply:
Done in 8a12d7a.